README.md

```diff
@@ -104,6 +104,31 @@ const rejectedTask = Task.resolve(new Error());
 const rejectedPromise = toPromise(rejectedTask);
 ```
 
+### `tap`
+
+The `tap` _function_ can be found in other libreries. It just takes a `callback` _function_ and return a function that calls the `callback` and _returns_ the _argument_. It's useful when you need to perform some side effect. Suppose you need to log the _resolved value_ at certain point of a `Task` _method chaining_. Without the `tap` _function_ you would probably do something like:
```

Comment:
libraries...
Too many functions... try to modify with: It takes a callback and returns a function that invokes the callback with the parameter and pass it trough